app/css/modals/community-launch.css

```diff
@@ -49,7 +49,7 @@
             @apply text-16 leading-huge;
```

Comment:
wouldn't you like to make this modal rounded? inside `& .--modal-content {}` rounded-16?